robot/impl/jobmanager_test.go

```diff
@@ -605,6 +609,201 @@ func TestJobManagerConfigChanges(t *testing.T) {
 	})
 }
 
+func TestJobManagerRemote(t *testing.T) {
+	t.Parallel()
```

Comment:
> t.Parallel()

Any reason? This looks like one of the tests that might _not_ be good to run in parallel give its need to listen on some port.